### PR TITLE
hook invite link details confirmation to accept flow

### PR DESCRIPTION
**Pull request by AMarcedone**

### CODE REVIEW

**Comment by pzduniak** [comment]
Per CI you need to generate some frontend stuff. `yarn build-actions` should be enough, I think.

**Comment by zapu** [comment]
there's more:
```
                Error Trace:    team_seitan_invitelink_test.go:66
                                                        team_seitan_invitelink_test.go:14
                Error:          Received unexpected error:
                                protocol not found: keybase.1.teamsUi
                Test:           TestTeamInviteSeitanInvitelinkHappy
    test_logger.go:64: TEST FAILED: TestTeamInviteSeitanInvitelinkHappy
```

File: go/teams/seitan_invitelink.go [hunk 1]
```diff
@@ -122,3 +122,49 @@ func GenerateInvitelinkURL(mctx libkb.MetaContext, ikey keybase1.SeitanIKeyInvit
...
+	teamName, err := keybase1.TeamNameFromString(resp.TeamName)
+	if err != nil {
+		return info, err
+	}
+
+	return keybase1.InviteLinkDetails{
+		InviteID:          inviteID,
+		InviterUID:        resp.InviterUID,
+		InviterUsername:   resp.InviterUsername,
+		InviterResetOrDel: resp.InviterResetOrDel == 1,
```
Comment (zapu) [inline go/teams/seitan_invitelink.go:163]: is this servertrust? I don't think that's useful

Comment (zapu) [reply go/teams/seitan_invitelink.go:163]: well maybe it is. protects from account takeover but not from evil keybase.

**Pull request merged**